backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -231,6 +246,12 @@ def test_qnn_backend_mean_dim(self):
             with self.subTest(i=i):
                 self.lower_module_and_test_output(module, sample_input)
 
+    @unittest.skip("it will hang in runtime")
+    def test_qnn_backend_mha(self):
+        module = MultiheadAttention()  # noqa: F405
```

Comment:
is this meant to test both linear and sdpa layers in MHA?

Reply:
No, we just test the functionality of MHA.
https://github.com/CodeLinaro/executorch/blob/2dc87fe8352299e77701ab8e9019201f545a9a1b/backends/qualcomm/tests/models.py#L297